pkgs/applications/science/biology/gatk/default.nix

```
@@ -8,10 +8,18 @@ stdenv.mkDerivation rec {
     sha256 = "0hjlsl7fxf3ankyjidqhwxc70gjh6z4lnjzw6b5fldzb0qvgfvy8";
   };
 
+  nativeBuildInputs = [ makeWrapper ];
+
+  dontUnpack = true;
+
   installPhase = ''
     mkdir -p $out/bin
-    install -m755 -D $src/gatk $out/bin/
-    install -m755 -D $src/gatk-package-${version}-local.jar $out/bin/
+    cp $src/gatk-package-${version}-local.jar $out/bin/
+    cp $src/gatk-package-${version}-spark.jar $out/bin/
+    cp $src/gatk $out/bin/
+  '';
+  postFixup = ''
+    wrapProgram $out/bin/gatk --prefix PATH : ${lib.makeBinPath [ jre python3 ]}
   '';
```

```suggestion
  buildInputs = [ python3 ];

  installPhase = ''
    install -dm755 -D $src/gatk-package-${version}-local.jar $out/bin/
    install -dm755 -D $src/gatk-package-${version}-spark.jar $out/bin/
    install -dm755 -D $src/gatk $out/bin/
  '';

  postFixup = ''
    wrapProgram $out/bin/gatk --prefix PATH : ${lib.makeBinPath [ jre ]}
  '';
```
That should already be enough.